nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -623,9 +623,8 @@ define([
                     app_id: appSpecInfo.id,
                     tag: appSpecInfo.tag || 'release',
                     version: appSpecInfo.git_commit_hash,
-                    params: params[fileType].filePaths.map((filePathParams) => {
-                        return Object.assign({}, filePathParams, params[fileType].params);
-                    }),
+                    params: params[fileType].filePaths,
+                    shared_params: params[fileType].params,
```

Comment:
Does this return a dict or a dict wrapped in a list?

Reply:
a dictionary